tests/integration/defs/disaggregated/test_ad_disagg.py

```
@@ -0,0 +1,1053 @@
+            ).outputs[0]
+            print(f"[AD DISAGG TEST] generation output: {response_summary([generation_output])}")
+
+    return {
+        "context": context_output,
+        "generation": generation_output,
+    }
+
+
+async def run_async_requests(llm, prompts, sampling_params_kwargs, disaggregated_params):
+    futures = []
+    for prompt, params in zip(prompts, disaggregated_params):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)